release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2833,3 +2833,29 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.13.0 --pullspecs
 ----
 //replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+[id="ocp-4-13-1"]
+=== RHSA-2023:3304 - {product-title} 4.13.1 bug fix and security update
+
+Issued: 2023-05-30
+
+{product-title} release 4.13.1, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:3304[RHSA-2023:3304] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3303[RHSA-2023:3303] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.1 --pullspecs
+----
+
+[id="ocp-4-13-1-bug-fixes"]
+==== Bug fixes
+* Previously, assisted installations may have encounter a transient error, and if that occurred the installation failed to recover. With this update, transient errors are retried correctly. (link:https://issues.redhat.com/browse/OCPBUGS-13138[*OCPBUGS-13138*])
+
+* Previously, oc-mirror OpenShift CLI (`oc`) plugin would fail with a `401 unauthorized` error for some registries when nested path exceeds the expected maximum path-components. With this update, the default `--max-nested-paths` flag has had the integer set to 0 or no limit. As a result, the generated `ImageContentSourcePolicy` will contain source and mirror references up to the repository level as opposed to the namespace level used by default. (link:https://issues.redhat.com/browse/OCPBUGS-13138[*OCPBUGS-13138*])
```

Comment:
```suggestion
* Previously, oc-mirror OpenShift CLI (`oc`) plugin would fail with a `401 unauthorized` error for some registries when a nested path exceeded the expected maximum path-components. With this update, the default integer of the `--max-nested-paths` flag is set to either 0 or no limit. As a result, the generated `ImageContentSourcePolicy` will contain source and mirror references up to the repository level as opposed to the namespace level used by default. (link:https://issues.redhat.com/browse/OCPBUGS-13138[*OCPBUGS-13138*])
```

🔍 "path-components"?

Does 0 == "no limit", or is it something else?

Reply:
0 = no limit so I incorporate the necessary change. Thanks, Max